<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Downloads/appIcon.ico</normaloff>../Downloads/appIcon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #FFF;
	padding-right: 15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { 
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Зашифровка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Расшифровка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(240,248,255)
}</string>
    </property>
    <property name="text">
     <string>Шифр Цезаря</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #FFF;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Расшифровка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(240,248,255)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Аффиный шифр&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61)
}</string>
    </property>
    <property name="text">
     <string>Зашифровка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Расшифровка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61)
}</string>
    </property>
    <property name="text">
     <string>Зашифровка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: rgb(240,248,255)
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Шифр Хилла&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>370</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: #FFF;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ввод вручную</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ввод из файла</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>553</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
     background-color: rgb(119,221,231)
 }</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ввод вручную</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ввод из файла</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Ввод вручную</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ввод из файла</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>158</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFF;
	background-color: rgb(30,33,61);
}</string>
    </property>
    <property name="text">
     <string>Выберите файл</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	border: 2px solid rgb(37, 39, 48);
	color: #FFF;
	background-color: rgb(34, 36, 44);
}
QTextEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QTextEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
